app/models/news_article.rb

```diff
@@ -4,11 +4,33 @@ class NewsArticle < Node
            source_type: 'Section'
 
   alias_method :publisher, :section
-
   store_attribute :data, :release_date, Date
 
-  validates_presence_of :section, :parent
-  validates :parent, ancestry_depth: { minimum: 2 }
+  friendly_id :date_and_name, use: [:slugged, :scoped], routes: :default, scope: :section
+
+  validates_presence_of :section
 
   scope :published, -> { where state: 'published' }
+
+  def layout
+    'news_article'
+  end
+
+  # News Articles require specific redirects to a custom route that
+  # is not defined by node hierarchy. As such, this method is a convenience
+  # wrapper around the url helpers, to allow this model to override the default
+  # redirect action.
+  def full_path
```

Comment:
I guess it just feels wrong to be calling routing aspects of Rails inside an ActiveRecord model. 

It's not a big deal, maybe since we're trying to move fast, let's just push this through then if I can think of a clean refactor I will raise as a separate PR...